paddle/operators/functor/CMakeLists.txt

```diff
@@ -0,0 +1,5 @@
+if(WITH_GPU)
```

Comment:
We have some operations on tensor in our code internally. Now, the internal operations are in paddle/operators/math directory, and there are some global functions. If functor make codes cleaner, we can use functor instead of function(Just like TensorFlow does).